op-node/rollup/chain_spec.go

```diff
@@ -55,3 +61,19 @@ func (s *ChainSpec) MaxRLPBytesPerChannel(t uint64) uint64 {
 	}
 	return maxRLPBytesPerChannelBedrock
 }
+
+// IsFeatMaxSequencerDriftConstant specifies in which fork the max sequencer drift change to a
+// constant will be performed.
+func (s *ChainSpec) IsFeatMaxSequencerDriftConstant(t uint64) bool {
```

Comment:
Do we want to expose this as part of the public api? Seems like callers care more about the value itself

Reply:
I see, its for testing purposes

Reply:
I'm fine with this here. I think generally I lean to removing the feature specific IsActive functions in favor of always activating it with the network upgrade once the network upgrade feature set is finalized.

Reply:
Yes we can do that, a simple find-replace once the feature set is frozen, plus removing the `IsFeat...` method. This direction is much easier than the other, i.e., taking something out of a fork, because you'd need to consider every single usage of `Is<Fork>` individually.

I'll leave it in this PR, and later remove it in a follow up.